docs/adding-zones.md

```diff
@@ -1,33 +1,30 @@
+This document explains how to add zones to an existing MinIO Tenant with Operator. This is only applicable to a Tenant (MinIO Deployment) created by MinIO Operator.
 
 Read more about MinIO Zones design in [MinIO Docs](https://github.com/minio/minio/blob/master/docs/distributed).
 
 ## Getting Started
 
-Assuming you have a MinIO cluster with single zone, `zone-0` with 4 drives (as shown in [examples](https://github.com/minio/operator/tree/master/examples)). You can dd a new zone `zone-1` with 4 drives using `kubectl patch` command.
+Assuming you have a MinIO Tenant with single zone, `zone-0` with 4 drives (as shown in the document [here](https://github.com/minio/operator/blob/master/README.md)). You can add a new zone `zone-1` with 4 drives using `kubectl patch` command.
 
 ```
 kubectl patch tenants.minio.min.io minio --patch "$(cat examples/patch.yaml)" --type=merge
 ```
 
 If you're using a custom configuration (e.g. multiple zones or higher number of drives per zone), make sure to change `patch.yaml` accordingly.
 
-**NOTE**: Important points to consider _before_ using cluster expansion:
+**NOTE**: Important points to consider _before_ using Tenant expansion:
 
-- During cluster expansion, MinIO Operator removes the existing StatefulSet and creates a new StatefulSet with required number of Pods. This means, there is a short downtime during expansion, as the pods are terminated and created again. As existing StatefulSet pods are terminated, its PVCs are also deleted. It is _very important_ to ensure PVs bound to MinIO StatefulSet PVCs are not deleted at this time to avoid data loss. We recommend configuring every PV with reclaim policy [`retain`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retain), to ensure the PV is not deleted. If you attempt cluster expansion while the PV reclaim policy is set to something else, it may lead to data loss. If you have the reclaim policy set to something else, change it as explained in [Kubernetes documents](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy/).
+- During Tenant expansion, MinIO Operator removes the existing StatefulSet and creates a new StatefulSet with required number of Pods. This means, there is a short downtime during expansion, as the pods are terminated and created again. As existing StatefulSet pods are terminated, its PVCs are also deleted. It is _very important_ to ensure PVs bound to MinIO StatefulSet PVCs are not deleted at this time to avoid data loss. We recommend configuring every PV with reclaim policy [`retain`](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#retain), to ensure the PV is not deleted. If you attempt Tenant expansion while the PV reclaim policy is set to something else, it may lead to data loss. If you have the reclaim policy set to something else, change it as explained in [Kubernetes documents](https://kubernetes.io/docs/tasks/administer-Tenant/change-pv-reclaim-policy/).
```

Comment:
Nit: I might remove `short` and just state 'downtime during expansion', as that's subjective (one customer's definition of "short" might not match another, and that can cause some trouble). This might be a good place to remind users that:

```
The Tenant expansion process requires removing the existing `StatefulSet` and creating a new `StatefulSet` with the required number of pods. Kubernetes automatically terminates and re-creates pods and `PVCs` during this process. Since Minio requires at least `(Volumes/2)+1` volumes to support regular read and write operations, the expansion process may result in a period of downtime where MinIO returns errors for read and write operations.
```
Or something along those lines.